models/sweeps/add-w-and-b-to-your-code.mdx

```diff
@@ -180,7 +96,7 @@ Next, add the following to the code example:
 
 1. Import the W&B Python SDK (`wandb`) and PyYAML (`yaml`). PyYAML is used to read in our YAML configuration file.
 2. Read in the configuration file.
-3. Use the [`wandb.init()`](/models/ref/python/functions/init) API to generate a background process to sync and log data as a [W&B Run](/models/ref/python/experiments/run). Pass the config object to the config parameter.
+3. Use [`wandb.init()`](/models/ref/python/functions/init) to generate a background process to sync and log data as a [W&B Run](/models/ref/python/experiments/run). Pass the config object to the config parameter.
```

Comment:
Is "generate" the right word for a background process, or something else like "start" or "spawn"?